ord_schema/validations.py

```diff
@@ -40,6 +40,8 @@ class ValidationOptions:
     validate_ids: bool = False
     # Require ReactionProvenance for Reactions.
     require_provenance: bool = False
+    # Allow reactions with valid reaction SMILES and nothing else.
+    allow_reaction_smiles_only = True
```

Comment:
For consistency, use typing here as well?

Reply:
ah thanks; done